README.md

```diff
@@ -47,7 +47,9 @@
 
 ### Python
 
-- [fastparquet](https://github.com/dask/fastparquet/) - A Python implementation of the Parquet columnar file format. 
+- [fastparquet](https://github.com/dask/fastparquet/) - A Python implementation of the Parquet columnar file format.
+- [petastorm](https://github.com/uber/petastorm) - Petastorm is a library enabling the use of Parquet storage from Tensorflow, Pytorch, and other Python-based ML training frameworks.
+- [dask](https://github.com/dask/dask) - Dask is a flexible parallel computing library for analytics that can efficiently load and process multiple Parquet files as a unified dataset, enabling distributed computations on datasets larger than memory.
```

Comment:
I would not include dask, as it's one of the many libraries based on PyArrow (see a list here: https://huggingface.co/docs/hub/datasets-libraries) + it's not specific to Parquet.

Note that it also supports another engine, dask-cudf, based on cudf, and I think we should include cudf in the "Multiple languages" section as it has its own native support for Parquet.
Note 2: see https://parquet.apache.org/docs/file-format/implementationstatus/ for other implementations that might be missing.

Reply:
or maybe we add dask , but also pandas... As you prefer

Reply:
@severo I agree with you, dask is not specific to Parquet! And there are so many similar tools that can read some Parquet files... I guess that I just added it is because there already was spark and polars.

Reply:
yes: I added spark and polars because they have their own implementation